refresh-example.sh

```diff
@@ -37,9 +37,11 @@ fi
 \rm -fr example
 
 # Make the new example
-npx react-native init example
+npm_config_yes=true npx react-native init example --skip-install
```

Comment:
npm_config_yes=true makes npx run non-interactively after they added a  new prompt confirming you really want to download and execute internet code